nx.json

```diff
@@ -47,16 +47,14 @@
 				"core",
 				"tools",
 				"!{projectRoot}/stories/*.js",
-				"{workspaceRoot}/tasks/mod-extractor.js",
-				{ "externalDependencies": ["gulp", "postcss"] },
+				{ "externalDependencies": ["postcss"] },
 				{ "env": "NODE_ENV" }
 			],
 			"outputs": ["{projectRoot}/dist", "{projectRoot}/metadata/mods.md"],
 			"executor": "nx:run-commands",
 			"options": {
 				"commands": [
-					"gulp --gulpfile {projectRoot}/gulpfile.js",
-					"node ./tasks/mod-extractor.js {projectRoot}"
+					"node -e 'require(\"./tasks/component-builder.js\").default()'"
```

Comment:
I can pull this into a command line tool eventually but this is the "quick & dirty" way to access the script.